lib/helpers/DataTypeApiHelper.ts

```diff
@@ -123,7 +123,7 @@ export class DataTypeApiHelper {
     return await response.json();
   }
 
-  async createFolder(name: string, id?: string, parentId?: string) {
+  async createFolder(name: string, parentId?: string, id?: string) {
```

Comment:
Will we ever need to insert the id for the dataTypeFolder? This method even returns the id

Reply:
Sounds good!